Sources/Kevin/Accounts/KevinAccountsConfiguration.swift

```diff
@@ -11,24 +11,33 @@ import Foundation
 public class KevinAccountsConfiguration {
     
     let callbackUrl: URL
-    
+    let showUnsupportedBanks: Bool
+
     private init(
-        callbackUrl: URL
+        callbackUrl: URL,
+        showUnsupportedBanks: Bool
     ) {
         self.callbackUrl = callbackUrl
+        self.showUnsupportedBanks = showUnsupportedBanks
     }
     
     public class Builder {
         
         private let callbackUrl: URL
-        
-       public init(callbackUrl: URL) {
+        private let showUnsupportedBanks: Bool
```

Comment:
Builder should not have a constructor parameter with `showUnsupportedBanks` as only `callbackUrl` is mandatory. Provide this param in the builder instead like Android does [here](https://github.com/getkevin/kevin-android/pull/80/files). Default value for `showUnsupportedBanks` should be false.